### PR TITLE
feat: lift time to see to the backend

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

see #13927 

## Changes

* lifts all the time to see queries to the backend
* adds time to see data to data table node options

time-t-see

## How did you test this code?

👀

### CODE REVIEW

File: frontend/src/queries/query.ts [hunk 1]
```diff
@@ -141,35 +141,32 @@ export async function query<N extends DataNode = DataNode>(
...
+                ...queryNode,
+                teamId: queryNode.teamId ?? getCurrentTeamId(),
+                sessionId: queryNode.sessionId ?? currentSessionId(),
+                sessionStart: queryNode.sessionStart ?? now().subtract(1, 'day').toISOString(),
+                sessionEnd: queryNode.sessionEnd ?? now().toISOString(),
+            },
+            methodOptions
+        )
+    } else if (isTimeToSeeDataSessionsNode(queryNode)) {
+        return query(queryNode.source)
```
Comment (pauldambra) [inline frontend/src/queries/query.ts:165]: renamed the variable to `queryNode` so I could recurse here

File: frontend/src/queries/nodes/DataTable/dataTableLogic.ts [hunk 2]
```diff
@@ -113,6 +113,11 @@ export const dataTableLogic = kea<dataTableLogicType>([
                         }
                     }
                 }
+
+                if (response && sourceKind === NodeKind.TimeToSeeDataSessionsQuery) {
+                    return (response as Record<string, any>[]).map((row) => ({ result: row }))
+                }
```
Comment (mariusandra) [inline frontend/src/queries/nodes/DataTable/dataTableLogic.ts:119]: Suggestion: perhaps let's start typing these as `as NonNullable<TimeToSeeDataSessionsQuery['response']>`?

File: frontend/src/queries/nodes/DataTable/renderColumn.tsx [hunk 3]
```diff
@@ -182,6 +182,9 @@ export function renderColumn(
                 {String(value)}
             </CopyToClipboardInline>
         )
+    } else if (key.startsWith('user.') && isTimeToSeeDataSessionsQuery(query.source)) {
+        const [parent, child] = key.split('.')
+        return typeof record === 'object' ? record[parent][child] : 'unknown'
```
Comment (mariusandra) [inline frontend/src/queries/nodes/DataTable/renderColumn.tsx:187]: General comment for this file, for sure out of scope here: This `renderColumn` is getting rather large. We should probably split it up by query type.

**Review by mariusandra — APPROVED** [review]
Looks good to me. A typing suggestion inline, but ✅

**Review by mariusandra — APPROVED** [review]

**Pull request merged**